kinto_webpush/listener.py

```diff
@@ -0,0 +1,48 @@
+import re
+import requests
+
+from kinto.core.utils import instance_uri
+from six import iteritems
+
+
+def on_resource_changed(event):
+    """
+    Everytime an object is created/changed/deleted, we update the
+    bucket counters.
+
+    If a new object exceeds the quotas, we reject the request.
+    """
+    payload = event.payload
+    action = payload['action']
+    resource_name = payload['resource_name']
+    event_uri = payload['uri']
+
+    permission = (action == "read") and "read" or "write"
```

Comment:
Note: in the blueprint, this was called *operation*. Why create and delete are considered as write?